src/EditorFeatures/TestUtilities/Diagnostics/TestDiagnosticAnalyzerDriver.cs

```diff
@@ -72,9 +72,19 @@ private async Task<IEnumerable<Diagnostic>> GetDiagnosticsAsync(DiagnosticAnalyz
             return allDiagnostics;
         }
 
+        private bool areSpansCompatible(TextSpan firstSpan, TextSpan secondSpan, bool shouldVerifyEntireSpan)
```

Comment:
>areSpansCompatible [](start = 21, length = 18)

Nit: Either should be a local function OR the method name must start with a capital letter and must be a static method.